src/test/java/programs/string/binarySearchIntTest.java

```diff
@@ -0,0 +1,16 @@
+package programs.string;
+
+import org.junit.Test;
+import programs.BinarySearchInt;
+
+import static org.junit.Assert.assertEquals;
+
+public class binarySearchIntTest {
```

Comment:
Name of the class, binarySearchIntTest -> BinarySearchIntTest. Please update this before merging.